adapters/freewheelssp/freewheelssp_test.go

```diff
@@ -18,3 +22,79 @@ func TestJsonSamples(t *testing.T) {
 
 	adapterstest.RunJSONBidderTest(t, "freewheelssptest", bidder)
 }
+
+func TestFreewheelAdapter_MakeBids(t *testing.T) {
```

Comment:
This is not how you should test your code. Please create json test cases. Refer - https://github.com/prebid/prebid-server/blob/2cc2c90a09708de6fe5cdbf5d798a08db0b806d9/adapters/freewheelssp/freewheelssptest/exemplary/multi-imp.json#L3